docs/local_setup.md

```diff
@@ -12,7 +12,8 @@ This software easily allows integration with GitHub version control and seamless
 
 Git Bash is a terminal that we use to rectify Unix line endings within windows machines. Git should have been installed with GitHub Desktop. If not, follow the steps below.
 
-Follow the link here to download Git. Once GitBash is installed, use the following command: `git config --global core.autocrlf false`
+Follow the link here to download Git. Once GitBash is installed, use the following command within that console: `git config --global core.autocrlf false`
+Additionally open Github Desktop and enter the following lines: `git config core.eol lf', 'git config core.autocrlf "input"`.
```

Comment:
Two nitpick-y bits here:
- would add some wording to make it clear this optional and only if you use github desktop
- a beginning user won't know where in github desktop to enter the following lines. going to have to fill in that middle of step of "go to whatever menu it's called -> select terminal (or whatever they call it in the menu)"